modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ServerConfigurator.java

```diff
@@ -141,8 +143,9 @@ public String getContextName() {
      */
     protected void parseAndCopy(Properties config) throws IOException, TemplateException, ConfigurationException {
         // FilenameFilter for excluding "nuxeo.defaults" files from copy
+        String nuxeoEnvironmentConf = generator.getNuxeoEnvironmentConfName();
```

Comment:
this is only used once line 148, is this for clarity ?

Reply:
What do you mean?

Reply:
The variable is defined here and seems to be used only once right after on line 148 so I would have just made a call to `generator.getNuxeoEnvironmentConfName();` there instead of defining a variable. Unless you did this to make the call shorter and clearer on line 148 ?

Reply:
If I correctly remember, the line you're talking about is a function and it is called several times. That's the reason we get the variable before, otherwise, we will call it several times.

Note: for future reviews, comment on line(s) you want to talk about.